scripts/globals/effects/level_restriction.lua

```diff
@@ -10,6 +10,20 @@ effectObject.onEffectGain = function(target, effect)
         target:levelRestriction(effect:getPower())
         target:messageBasic(xi.msg.basic.LEVEL_IS_RESTRICTED, effect:getPower()) -- <target>'s level is restricted to <param>
         target:clearTrusts()
+
+        if xi.settings.map.DESPAWN_JUGPETS_BELOW_MINIMUM_LEVEL then
+            local pet = target:getPet()
+            local masterLevel = target:getMainLvl()
+            print(pet)
```

Comment:
This debug print can most likely be removed

Reply:
Oops, fixed.